grabbit/src/main/resources/META-INF/spring/client-batch-job.xml

```diff
@@ -33,12 +33,18 @@
 
     <import resource="client-batch-job-context.xml"/>
 
+    <import resource="client-validateJob.xml" />
     <import resource="client-namespace-step.xml" />
     <import resource="client-jcrNodes-step.xml" />
     <import resource="client-deleteBeforeWrite-steps.xml" />
 
     <batch:job id="clientJob" xmlns="http://www.springframework.org/schema/batch" job-repository="clientJobRepository">
 
+        <batch:decision id="validateJob" decider="validJobDecider">
+            <next on="VALID" to="clientWorkflowOff" />
+            <fail on="INVALID" exit-code="VALIDATION_FAILED" />
+        </batch:decision>
+
```

Comment:
Why would you expect those paths to fail?  I'm sure you have "/content" defined on author

It would fail for a path such as, "/content/foo/a", where "/content/foo" is not yet defined on author.